Meshtastic/Views/Nodes/MeshMap.swift

```diff
@@ -33,6 +33,7 @@ struct MeshMap: View {
 	@Namespace var mapScope
 	@State var mapStyle: MapStyle = MapStyle.standard(elevation: .flat, emphasis: MapStyle.StandardEmphasis.muted, pointsOfInterest: .excludingAll, showsTraffic: false)
 	@State var position = MapCameraPosition.automatic
+	@State private var distance = 10000.0
```

Comment:
I think this initial value will only matter when someone tries to deep link to a node before they've zoomed into an area on this map. (initial map zoom is set separately by `MapCameraBounds(..., maximumDistance: .infinity),`

It might be nice to tune this to whatever we find to be the most common ranges are when someone's wanting to view a point of interest. (be it waypoint or node)